src/_locales/fr/messages.json

```
@@ -225,6 +225,26 @@
     "description": "The context menu entry shown for generating QR codes from a selected link with an access key.",
     "hash": "312595c39b5cbc8edf0baab8c16875af"
   },
+  "contextMenuSaveImageCanvas": {
+    "message": "Sauvegarder QR-Code comme image…",
+    "description": "The context menu entry shown for saving PNG images (from a canvas) in the popup.",
+    "hash": "885e0352965b98e4b3d0aa8f09d4524b"
+  },
+  "contextMenuSaveImageCanvasAccessKey": {
+    "message": "Sauvegarder QR-Code comme &image…",
```

```suggestion
    "message": "Enregistrer le QR-Code en tant que &image…",
```
I don't know what is supposed to be the '&' so maybe "que" should be replaced with "qu'".